internal/cli/terraform.go

```diff
@@ -187,3 +222,50 @@ import {
 
 	return t.Execute(file, data)
 }
+
+func generateTerraformResourceConfig(ctx context.Context, outputDIR string) error {
+	absoluteOutputPath, err := filepath.Abs(outputDIR)
+	if err != nil {
+		return err
+	}
+
+	installer := &releases.ExactVersion{
+		Product:    product.Terraform,
+		Version:    version.Must(version.NewVersion("1.5.0")),
+		InstallDir: absoluteOutputPath,
+	}
+
+	execPath, err := installer.Install(ctx)
+	if err != nil {
+		return err
+	}
+
+	tf, err := tfexec.NewTerraform(absoluteOutputPath, execPath)
+	if err != nil {
+		return err
+	}
+
+	if err = tf.Init(context.Background()); err != nil {
+		return err
+	}
+
+	// -generate-config-out flag is not supported by terraform-exec, so we do this through exec.Command.
+	cmd := exec.CommandContext(ctx, execPath, "plan", "-generate-config-out=generated.tf")
+	cmd.Dir = absoluteOutputPath
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Printf("%s", output)
```

Comment:
Do we necessarily need or want to print this output? This command might benefit from not bleeding out the implementation here. Or in other words, it has a chance of being magic.

Reply:
Updated in [`0117602` (#796)](https://github.com/auth0/auth0-cli/pull/796/commits/01176024dfea4bf8c64d41558b234b9e7bb142e6).  But now if there's any failure from the terraform init or terraform plan commands we won't be giving any visibility into what's happening, but simply default to letting the user run those commands manually.